src/index.ts

```diff
@@ -568,14 +573,16 @@ export async function assemble(config: Configuration, components: Components): P
     sharingDID: () => DID.sharing(components.crypto),
 
     // File system
-    loadFileSystem: (username: string) =>
-      loadFileSystem({ config, username, dependencies: components }),
-    recoverFileSystem: (params: RecoverFileSystemParams) =>
-      recoverFileSystem({
-        auth,
-        dependencies: { crypto: components.crypto, reference: components.reference, storage: components.storage },
-        ...params,
-      }),
+    loadFileSystem: (username: string) => loadFileSystem({ config, username, dependencies: components }),
+    recoverFileSystem: (params: RecoverFileSystemParams) => recoverFileSystem({ auth, dependencies: components, ...params }),
+
+    fileSystem: {
+      addPublicExchangeKey: (fs: FileSystem) => FileSystemData.addPublicExchangeKey(components.crypto, fs),
+      addSampleData: (fs: FileSystem) => FileSystemData.addSampleData(fs),
```

Comment:
What are the use cases where developer would want to use this shorthand? I could see where it could see it being useful for quick experimentation, but I don't think developers will want exactly the directories it creates.

If we are the primary consumer of the shorthand, we should consider leaving it buried to keep the top-level API as simple as possible.

Reply:
The public key ones would be for whenever you have an app that you want to receive a private share on. The behaviour of these functions will change over time (eg. rs-wnfs will keep exchange keys in the root).

The sample data one would be for apps like Drive, which is less useful, but doesn't hurt to have it there I guess, even if it's for quick experimentation.

Reply:
Ah yeah, the public key ones are good additions 👌. I was questioning `addSampleData`.

Adding `addSampleData` has some cost because it's another bit of API for developers to learn. If it has a good use case for them, then it's worth adding. Otherwise, we should consider leaving it out.

It seems like quick experimentation is a secondary use case. If we want to support the quick experimentation, maybe `addSampleData` could add a file in addition to directories?

Reply:
I'll add a file 👍